### ISSUE TITLE
Doesn't download all posts. Flagged as Duplicates

### ISSUE THREAD

**Issue by Joakimgreenday** [Avnsx/fansly#59]
There are posts that aren't downloaded, it also says 218 duplicates declined! Those 218 could be mistaken for a duplicate but they aren't those are missing posts photos and I'm pretty sure. 
Wouldn't it be good if there was an option in the config "Put duplicates in separate folder", but it still downloads the duplicates?

Either way it's missing posts!

**Comment by RickyLaFleur420** [comment]
Yeah, it seems to be downloading previews, but not the unlocked posts I can see on the site as a subscriber.

**Comment by Avnsx** [comment]
@Joakimgreenday 
> Those 218 could be mistaken for a duplicate

It's very unlikely to be mistaken for a duplicate, there's an actual [fool proof algorithm](https://github.com/JohannesBuchner/imagehash) that hashes the images and decides if they're very similar to each other or not.

> Either way it's missing posts!

What creator are you encountering this issue with? Which posts is it missing (subscriber or preview)?

**Comment by Avnsx** [comment]
@RickyLaFleur420 
> Yeah, it seems to be downloading preview images, but not the unlocked image posts I can see on the site as a subscriber.

I've never spent any money on NSFW content (or any at all) ever, so it's hard for me to debug issues like this.

If you're having trouble with subscribed posts not being downloaded and really want it to get fixed within fansly scraper; would you for debugging purposes provide me an account through e-mailing me credentials to ``AvnDev@protonmail.com``

Also I'm wondering if this issue started happening recently for you or if you're encountering this issue ever since the first use of fansly scraper.

**Comment by RickyLaFleur420** [comment]
> @RickyLaFleur420
> 
> > Yeah, it seems to be downloading preview images, but not the unlocked image posts I can see on the site as a subscriber.
> 
> I've never spent any money on NSFW content (or any at all) ever, so it's hard for me to debug issues like this.
> 
> If you're having trouble with subscribed posts not being downloaded and really want it to get fixed within fansly scraper; would you for debugging purposes provide me an account through e-mailing me credentials to `AvnDev@protonmail.com`
> 
> Also I'm wondering if this issue started happening recently for you or if you're encountering this issue ever since the first use of fansly scraper.

This is a brand new issue, I'd never seen this before today - I noticed it downloaded the preview image but not the subscribed images.

**Comment by Avnsx** [comment]
You guys are sure this happens with every creator you subscribed to, right?

**Comment by Avnsx** [comment]
You guys are sure, subscriber only content can't be downloaded, with every creator you subscribed to, right?

Does this apply to subscribed content from messages only?

Or does it maybe apply to subscribed content from purchased bundles only?

@raucodes @Joakimgreenday @RickyLaFleur420

**Comment by wireman21** [comment]
I have a similar issue but the scraper does not download content from "messages"

**Comment by wireman21** [comment]
I have the same problem, but it doesn't download content in messages

**Comment by e2489** [comment]
Having the same issue. What it's doing for me is when a post has content for multiple tiers, it downloads the content from the lowest tier, but not the highest (which I'm subscribed to and can see).

**Comment by Avnsx** [comment]
If anyone lends me their account, which has currently access to any paid bundles and messages and what not, I can fix these issues really quickly and also add that one feature y'all been requesting for a while called PPV content (https://github.com/Avnsx/fansly/issues/12).  Send an e-mail to ``AvnDev@protonmail.com``

I'm not planning to spend money on nsfw content anytime soon!

**Comment by aero1313** [comment]
@Avnsx Do you still need an account with access to paid content?

**Comment by Avnsx** [comment]
> @Avnsx Do you still need an account with access to paid content?

Yes, still

**Comment by aero1313** [comment]
Just sent you an email.

**Comment by jeffwhite6557** [comment]
First attempt to use fansly scraper v0.3.6.  I got it to run by manually configuring config.ini.  I like what I see except it did not download media from the messages directory.  I know there is 2 pics and 1 video that should be downloadable.  Is there any thing I can try to fix this?

**Comment by Avnsx** [comment]
> Is there any thing I can try to fix this?
Unless you're subscribed to a creator on fansly and know how to program in python, no not really

**Comment by jeffwhite6557** [comment]
Thanks for the reply.  I'll bid my time for you to get to it.

**Comment by kervala** [comment]
I also have the same issue :) All previews have been downloaded correctly, but when I subscribe a creator, it only downloads some posts :(

I noticed something else, unlocked posts with the largest size are in fact JPEG files (they have FF D8 FF DB header), but in timeline JSON, they are declared as "image/png" :)

**Comment by frankgel1652** [comment]
Same here. All content from messages were downloaded, but got only a couple preview videos. Must've been from the lowest tier that only gives previews, even though I subbed the the tier above it, like what was mentioned previously.

**Comment by UpAndDown666** [comment]
Hi all, I have never tried that content didn't get downloaded (apart from when it was already downloaded), but I have encountered some content that was marked as preview. I also encountered once where the content was posted both as preview and locked, and in that case the preview gets downloaded first, which was quite confusing.

**Comment by Avnsx** [comment]
Most likely fixed through https://github.com/Avnsx/fansly/pull/68/commits/d7da87dacbdf37fee87f886dbe73357a6b505eaf & https://github.com/Avnsx/fansly/commit/b30a65df220f13b95cba5fa4e5b032baa82a1d75

Currently only works through the python version though, so let me know what you think of it before I release a compiled version of the fansly scraper update later this week

**Comment by BobafettX** [comment]
I'm using 0.3.6 and it works getting subscribed videos both in messages and timeline.

**Comment by crunk323** [comment]
I'm using 0.3.5 and it's not downloading posts posted after 5/10/23. Only all the ones on that date and before that date.

**Comment by Avnsx** [comment]
> I'm using 0.3.5 and it's not downloading posts posted after 5/10/23. Only all the ones on that date and before that date.

I'm aware of this and have already locally implemented a fix for this in my developer version, which I'll ship within a couple days in 0.3.6

It's caused by fansly introducing a new media streaming format called .m3u8 and this format is served a little differently, which breaks the older versions of the scraper.

Relevant: https://github.com/Avnsx/fansly/issues/80

**Comment by Avnsx** [comment]
Should be fixed with https://github.com/Avnsx/Fansly-Downloader-App/commit/8c591cb2d1d1ee35a1243f788d1cb83c99e65209

Would be happy if you guys cloned the repository againd and tried that version out through python and told me if it's free of bugs.

Will release that 0.4 version as compiled executable within a couple days, once people start providing some feedback.

**Comment by Zolgonark** [comment]
Any update on the exe version for 0.4? I'd test it in python to give feedback, but I dont have any clue on how it works :(
